docs/byoh-instance-pre-requisites.md

```diff
@@ -4,6 +4,7 @@ The following pre-requisites must be fulfilled in order to add a Windows BYOH no
 * The Docker container runtime must be installed on the instance.
 * The instance must be on the same network as the Linux worker nodes in the cluster.
 * Port 22 must be open and running [an SSH server](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).
```

Comment:
Moving forward, would be nice to have/maintain just one doc to [set-up SSH in Windows](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/vsphere-golden-image.md#3-set-up-ssh)